adminapi/filters.py

```diff
@@ -245,11 +234,8 @@ class InsideOnlyNetwork(InsideNetwork):
 
 
 class Empty(BaseFilter):
-    def __repr__(self):
-        return 'Empty()'
-
-    def serialize(self):
-        return {'name': 'empty'}
+    def __init__(self):
+        self.value = None
```

Comment:
Wanna use the super function?
``` python
def __init__(self):
    super().__init__(None)
```
to not completely override but rather inherit the functionality?

Reply:
I think it is better to keep it simple.  Filters are not something that would evolve.